pkg/client_data/lib/account_api.dart

```diff
@@ -49,6 +49,32 @@ class ClientSessionStatus {
   bool get isValid => expires != null && DateTime.now().isBefore(expires);
 }
 
+/// Account-specific information about a likes.
+@JsonSerializable()
+class PackageLikes {
+  List<PackageLike> likedPackages;
+  PackageLikes(this.likedPackages);
```

Comment:
Let's use named parameters annotated with `required`, that way it's easier to read when we create the objects. I think this is also what we do for most other data types.